test/unicode-spec.js

```diff
@@ -72,9 +111,14 @@ describe("unicode", function() {
             'ÆÇÐØÞßæçðøþ}').toParse();
     });
 
+    it("should build Latin-1 inside \\text{}", function() {
+        expect('\\text{ÀÁÂÃÄÅÈÉÊËÌÍÎÏÑÒÓÔÕÖÙÚÛÜÝàáâãäåèéêëìíîïñòóôõöùúûüýÿ' +
+            'ÆÇÐØÞßæçðøþ}').toBuild();
+    });
+
```

Comment:
This seems to be a duplicate of the above -- perhaps a merge typo?

Reply:
This is uses `.toBuild()` whereas the one above uses `.toParse()`.  https://github.com/Khan/KaTeX/issues/1126 should address this.